Dockerfile-alpine.template

```diff
@@ -9,12 +9,12 @@ COPY requirements.txt /tmp/
 
 RUN set -ex \
     && apk add --no-cache --virtual .build-deps \
-        --repository http://dl-cdn.alpinelinux.org/alpine/v3.9/main \
+        --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
```

Comment:
I had to v3.8 to v3.9 for the addition of Django 2.2.x to the matrix. Do you know why `edge` has to be used here?

Reply:
I think they were taken out of `edge` when we did https://github.com/azavea/docker-django/commit/6a5baa8d1c2a609dccd61c43952811624b83fc37, but now there are packages in `edge` that are more up-to-date than the packages in `v3.9`.

Reply:
I like not using `edge`. Mostly because I'm worried about transitive dependency resolutions sneaking in from there.